libs/src/api/user.js

```diff
@@ -345,7 +345,11 @@ class Users extends Base {
   }
```

Comment:
FYI - none of the libs code changes in this PR are related, and all are code refactors / nits. i'm game to just rip them out if you all want